backend/src/api/server.py

```
@@ -0,0 +1,82 @@
+    """
+    def __init__(self, host: str = "127.0.0.1", port: int = 8000):
+        self.host = host
+        self.port = port
+        self.app = FastAPI(title= "test server")
+        self.register_routes()
+        controller.subscribe(
+            modulename="server",
+            config_events={"server": None},
+            message_events={"rep": self.handle_rep_message}
+        )
+        self.app.state._messages_buffer = []#用于存储消息的缓冲区，同时要暴露在app.state里,以便路由能访问到
```

Ruff (0.14.10)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)